packages/component/src/Utils/downscaleImageToDataURL/downscaleImageToDataURLUsingWorker.ts

```diff
@@ -1,7 +1,7 @@
 import workerFunction from './downscaleImageToDataURLUsingWorker.worker';
 import { type WorkerJob, type WorkerReturnValue } from './WorkerJob';
 
-function createWorker(fn: Function | string): Promise<Worker> {
+function createWorker(fn: ((...args: any[]) => any) | string): Promise<Worker> {
```

Comment:
nit: avoid `any`. Can unknown be used instead of `any`?

Reply:
It works.